commands/monitoring/pipeline/completer/completer.go

```diff
@@ -0,0 +1,21 @@
+package completer
+
+import (
+	"context"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/completions"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+)
+
+func PipelineIDs() []string {
+	pipelines, _, err := client.Must().Monitoring.PipelinesApi.PipelinesGet(context.Background()).Execute()
+	if err != nil {
+		return nil
+	}
+	pipelinesConverted, err := json2table.ConvertJSONToTable("items", jsonpaths.ApiGatewayGateway, pipelines)
```

Comment:
is this PR ready for review? We need to add `jsonpaths` for the pipeline object, this is the APIGateway object.